src/model/EnvironmentType.ts

```diff
@@ -54,29 +75,32 @@ export default class EnvironmentType extends Ressource {
     );
   }
 
-  static createAccess(projectId: string, environmentTypeId: string, access: any) {
+  static createAccess(params: CreateAccessParams) {
+    const { projectId, environmentTypeId, email, role } = params
     const { api_url } = getConfig();
     const url = `${api_url}/projects/${projectId}/environment-types/${environmentTypeId}/access`;
     return request(url, "POST", {
-      email: access.email,
-      role: access.role
+      email,
+      role
     }).then(response => new ProjectAccess(response._embedded.entity, url));
   }
 
-  static updateAccess(projectId: string, environmentTypeId: string, access: any) {
+  static updateAccess(params: UpdateAccessParams) {
+    const { projectId, environmentTypeId, accessId, role } = params
     const { api_url } = getConfig();
     const url = `${api_url}/projects/${projectId}/environment-types/${environmentTypeId}/access/${
-      access.id
+      accessId
     }`;
     return request(url, "PATCH", {
-      role: access.role
+      role
     }).then(response => new ProjectAccess(response._embedded.entity, url));
```

Comment:
It would be great to type this promise.  You can also transform this function to use `async/await`

Reply:
same with the typing

Reply:
Yep, it looks good 🙇